rocAL/rocAL/include/decoders/audio/audio_decoder.h

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright (c) 2019 - 2023 Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
Copy right in first few files have been changed from 2019 - 2023 -> 2023
Please check and change the same if that is applicable here

Reply:
Those changes are not a part of this PR, Will check with fiona

Reply:
Rebased and this issue is resolved